common/components/AnkiDialog.tsx

```diff
@@ -602,8 +602,8 @@ const AnkiDialog = ({
     const handleTrack3TextChange = useCallback((text: string) => {
         setTrack3(text);
         setLastAppliedTimestampIntervalToText(undefined);
-    }, []);
-
+    }, []);   
+     
```

Comment:
The extra space will fail the Prettier formatting check on this repo. Easiest way to fix this is run Prettier on the file whenever you make a change.

Reply:
Whoops, fixed!